### PR TITLE
Bug 7318/v4

### PR DESCRIPTION
**Pull request by victorjulien**
## Ticket

If your pull request is related to a Suricata ticket, please provide
the full URL to the ticket here so this pull request can monitor
changes to the ticket status:

Redmine ticket:

### CODE REVIEW

File: tests/firewall/firewall-06-tls-sni-enforce/test.yaml [hunk 1]
```diff
@@ -31,6 +44,13 @@ checks:
     match:
       event_type: drop
 - filter:
```
[the remark this answers is not in the document]
Comment (victorjulien) [reply tests/firewall/firewall-06-tls-sni-enforce/test.yaml:46]: we can patch it up later

File: tests/tls-extra-alert/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+This test shows that Suricata generates an additional alert for TLS
+for the given PCAP which shouldn't be there.
+
+## PCAP
+
+Internal.
+
+## Related issues
+
+None so far. State: Trying to establish what's the issue.
```
Comment (inashivb) [inline tests/tls-extra-alert/README.md:12]: nit: There is a ticket now: https://redmine.openinfosecfoundation.org/issues/7318

**Review by catenacyber — APPROVED** [review]
Ok for me to go on even if I think #2173 is better

**Review by jasonish — APPROVED** [review]
Effectively tests the issues. For cleanups, can be done later.

**Comment by victorjulien** [comment]
Merged in #2175

**Pull request closed without merging**